CHANGES.rst

```diff
@@ -99,7 +99,7 @@ Changed
 - The interface reconfiguration script now sends information about different
   operations to :command:`syslog` for easier debugging. [drybjed_]
 
-- Network interfaces that require changes are deconfigured in reverse order to
+- Network interfaces that require changes are reconfigured in reverse order to
```

Comment:
Actually this is IMO correct - interfaces are brought down in reverse order that they are defined, and then brought back up in "normal" order.